### DISCUSSION TITLE
If the dll generated with out Media Foundation feature is not working on Windows Server 2016 Machine

### DISCUSSION THREAD

**Discussion by SolairajMariappan** [FreeRDP/FreeRDP#6737] [Q&A]
If the dll generated with out Media Foundation feature is not working on Windows Server 2016 Machine but It worked on windows 10 & windows server 2012 machines. Do we have any work around for this issue ?

**Comment by akallabeth** [comment]
Use a build with `openh264` or `ffmpeg` (or both) as `h264` backend and disable `MediaFoundation`

**Reply by SolairajMariappan** [reply]
Thank you for the quick reply, I will use the build as suggested and Update my result.
